doc/source/flexibility/sequence_table.rst

```diff
@@ -0,0 +1,118 @@
+.. _sequence_table:
+
+===============================================================================
+SEQUENCE_TABLE(n) Function
+===============================================================================
+
+|Percona Server| 8.0.20-11 adds the SEQUENCE_TABLE() function.
+
+A sequence of numbers can be defined as an arithmetic progression when the common difference between two consecutive terms is always the same. If the common difference is positive, the sequence grows in positive terms; if the common difference is negative, the sequence grows in negative terms.
+
+The function is an inline table-valued function. A single SELECT statement generates a multi-row result set. A scalar function returns a single value of a specific data type, which is defined in a ``RETURNS`` clause.
+
+The SEQUENCE_TABLE() and the `JSON_TABLE() <https://dev.mysql.com/doc/refman/8.0/en/json-table-functions.html>`_ functions are the only TABLE functions available in MySQL.
+
+The basic syntax is the following:
+
+* SEQUENCE_TABLE(n) [AS] alias
+
+    .. code-block:: mysql
+
+        SEQUENCE_TABLE(
+            n
+        ) [AS] alias
+    
+``n:`` The number of generated values.
+
+The MySQL syntax requires the ``alias`` in the SELECT statement. The rules to define an alias are defined `here <https://dev.mysql.com/doc/refman/8.0/en/identifiers.html>`_.
+
```

Comment:
Also, it needs to be mentioned before the examples section that `SEQUENCE_TABLE()` result set will have a single column with predefined name `value` of type `BIGINT UNSIGNED` that can be referenced in the `SELECT` expressions.
Not only can you do `SELECT * FROM SEQUENCE_TABLE(n) AS tt` but `SELECT <expr(value)> FROM SEQUENCE_TABLE(n) AS tt` as well.

Reply:
Updated.